### PR TITLE
"Laurence" intro text

### LINKED ISSUE #9: Laurence on profiles
Maybe add something similar to this to the intro / abstract:

_"COSE is not an end-end system with guaranteed interoperability. It has to be profiled to interoperate. It is designed to serve a huge range of use cases so it has a lot of options."_

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fix #9

### CODE REVIEW

File: draft-fossati-cose-profiles.md [hunk 1]
```diff
@@ -61,14 +60,20 @@ entity:
 
 --- abstract
 
-This document lays a set of rules for how to define COSE profiles.
+COSE (STD96) is not an end-end system with guaranteed interoperability.
```
Comment (thomas-fossati) [inline draft-fossati-cose-profiles.md]:
```suggestion
COSE (STD96) is not an end-to-end system with guaranteed interoperability.
```

File: draft-fossati-cose-profiles.md [hunk 2]
```diff
@@ -61,14 +60,20 @@ entity:
 
 --- abstract
 
-This document lays a set of rules for how to define COSE profiles.
+COSE (STD96) is not an end-to-end system with guaranteed interoperability.
+It is designed to serve a range of use cases and therefore it has a lot of options.
+In general, two COSE implementation that want to interoperate need to agree on which subset of COSE features they will be using.
```
Comment (thomas-fossati) [inline draft-fossati-cose-profiles.md]:
```suggestion
In general, two COSE implementations that want to interoperate need to agree on which subset of COSE features they will be using.
```

File: draft-fossati-cose-profiles.md [hunk 3]
```diff
@@ -61,14 +60,20 @@ entity:
 
 --- abstract
 
-This document lays a set of rules for how to define COSE profiles.
+COSE (STD96) is not an end-to-end system with guaranteed interoperability.
+It is designed to serve a range of use cases and therefore it has a lot of options.
+In general, two COSE implementations that want to interoperate need to agree on which subset of COSE features they will be using.
```
Comment (henkbirkholz) [inline draft-fossati-cose-profiles.md]:
```suggestion
In general, two COSE implementations that want to interoperate require an agreement on which subset of COSE features they will use.
```

File: draft-fossati-cose-profiles.md [hunk 4]
```diff
@@ -61,14 +60,20 @@ entity:
 
 --- abstract
 
-This document lays a set of rules for how to define COSE profiles.
+COSE (STD96) is not an end-to-end system with guaranteed interoperability.
+It is designed to serve a range of use cases and therefore it has a lot of options.
+In general, two COSE implementations that want to interoperate require an agreement on which subset of COSE features they will use.
+This document provides a set rules for defining COSE profiles and registers a new COSE header parameter for in-band signalling of profile information.
```
Comment (henkbirkholz) [inline draft-fossati-cose-profiles.md]:
```suggestion
This document provides a set of rules for specifying such agreements as a "COSE profiles" and registers a new COSE header parameter for in-band signalling profile information.
```

File: draft-fossati-cose-profiles.md [hunk 5]
```diff
@@ -61,14 +60,20 @@ entity:
...
 --- middle
 
 # Introduction
 
-This document lays a set of rules for how to define COSE {{-cose}} profiles.
+COSE {{-cose}} is not an end-end system with guaranteed interoperability.
+It is designed to serve a range of use cases and therefore it has a lot of options.
+In general, two COSE implementation that want to interoperate need to agree on which subset of COSE features they will be using.
 
+This document provides a set rules for defining COSE profiles and registers a new COSE header parameter for in-band signalling of profile information.
```
Comment (thomas-fossati) [inline draft-fossati-cose-profiles.md]:
```suggestion
COSE {{-cose}} is not an end-to-end system with guaranteed interoperability.
 It is designed to serve a range of use cases and therefore it has a lot of options.
 In general, two COSE implementations that want to interoperate require an agreement on which subset of COSE features they will use.

This document provides a set of rules for specifying such agreements as a "COSE profiles" and registers a new COSE header parameter for in-band signalling profile information.
```

File: draft-fossati-cose-profiles.md [hunk 6]
```diff
@@ -61,14 +60,20 @@ entity:
 
 --- abstract
 
-This document lays a set of rules for how to define COSE profiles.
+COSE (STD96) is not an end-to-end system with guaranteed interoperability.
+It is designed to serve a range of use cases and therefore it has a lot of options.
+In general, two COSE implementations that want to interoperate require an agreement on which subset of COSE features they will use.
+This document provides a set of rules for specifying such agreements as a "COSE profiles" and registers a new COSE header parameter for in-band signalling profile information.
```
Comment (thomas-fossati) [inline draft-fossati-cose-profiles.md]:
```suggestion
This document provides a set of rules for specifying such agreements as "COSE profiles" and registers a new COSE header parameter for in-band signalling profile information.
```

File: draft-fossati-cose-profiles.md [hunk 7]
```diff
@@ -61,14 +60,20 @@ entity:
...
 --- middle
 
 # Introduction
 
-This document lays a set of rules for how to define COSE {{-cose}} profiles.
+COSE {{-cose}} is not an end-to-end system with guaranteed interoperability.
+ It is designed to serve a range of use cases and therefore it has a lot of options.
+ In general, two COSE implementations that want to interoperate require an agreement on which subset of COSE features they will use.
 
+This document provides a set of rules for specifying such agreements as a "COSE profiles" and registers a new COSE header parameter for in-band signalling profile information.
```
Comment (thomas-fossati) [inline draft-fossati-cose-profiles.md]:
```suggestion
This document provides a set of rules for specifying such agreements as "COSE profiles" and registers a new COSE header parameter for in-band signalling profile information.
```

File: draft-fossati-cose-profiles.md [hunk 8]
```diff
@@ -61,14 +60,20 @@ entity:
 
 --- abstract
 
-This document lays a set of rules for how to define COSE profiles.
+COSE (STD96) is not an end-to-end system with guaranteed interoperability.
+It is designed to serve a range of use cases and therefore it has a lot of options.
+In general, two COSE implementations that want to interoperate require an agreement on which subset of COSE features they will use.
+This document provides a set of rules for specifying such agreements as "COSE profiles" and registers a new COSE header parameter for in-band signalling profile information.
```
Comment (letmaik) [inline draft-fossati-cose-profiles.md]:
```suggestion
This document provides a set of rules for specifying such agreements as "COSE profiles" and registers a new COSE header parameter for in-band signalling of profile information.
```

File: draft-fossati-cose-profiles.md [hunk 9]
```diff
@@ -61,14 +60,20 @@ entity:
...
 --- middle
 
 # Introduction
 
-This document lays a set of rules for how to define COSE {{-cose}} profiles.
+COSE {{-cose}} is not an end-to-end system with guaranteed interoperability.
+ It is designed to serve a range of use cases and therefore it has a lot of options.
+ In general, two COSE implementations that want to interoperate require an agreement on which subset of COSE features they will use.
 
+This document provides a set of rules for specifying such agreements as "COSE profiles" and registers a new COSE header parameter for in-band signalling profile information.
```
Comment (letmaik) [inline draft-fossati-cose-profiles.md]:
```suggestion
This document provides a set of rules for specifying such agreements as "COSE profiles" and registers a new COSE header parameter for in-band signalling of profile information.
```

**Review by henkbirkholz — APPROVED** [review]
approve content, assuming CI is okay

**Pull request merged**